hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/LogEventViewGenerator.java

```diff
@@ -32,16 +41,28 @@
     try {
       List<LogEventView> list = new ArrayList<>();
       for (LogEvent logEventRecord : logEvents.getLogEvents()) {
-        LogEventView build =
+        String attributes = convertAttributes(logEventRecord.getAttributes());
+        LogEventView logEventView =
             LogEventView.newBuilder()
                 .setSpanId(logEventRecord.getSpanId())
                 .setTraceId(logEventRecord.getTraceId())
                 .setTimestampNanos(logEventRecord.getTimestampNanos())
                 .setTenantId(logEventRecord.getTenantId())
-                .setAttributes(convertAttributes(logEventRecord.getAttributes()))
+                .setAttributes(attributes)
                 .setSummary(getSummary(logEventRecord.getAttributes()))
                 .build();
-        list.add(build);
+        if (!StringUtils.isEmpty(logEventRecord.getTenantId()) && null != attributes) {
+          logEventAttributeSizeGauge
+              .computeIfAbsent(
+                  logEventRecord.getTenantId(),
+                  v ->
+                      PlatformMetricsRegistry.registerGauge(
+                          LOG_EVENT_ATTRIBUTE_SIZE_METRIC,
+                          Map.of("tenantId", logEventRecord.getTenantId()),
+                          new AtomicInteger(0)))
+              .set(attributes.length());
```

Comment:
Do you think, we should set avg length? Will this override the previous value before it was fetched by Prometheus?

Reply:
cc : @laxmanchekka

Reply:
It would basically report the observed value.